core/src/test/java/com/netease/arctic/io/TaskWriterTest.java

```diff
@@ -62,6 +75,39 @@ public void testBasePosDeleteWriter() throws IOException {
     Assert.assertEquals(1, result.size());
   }
 
+  @Test
+  public void testBasePosDeleteWriterMetrics() throws IOException {
```

Comment:
It seems new test case cover old `testBasePosDeleteWriter` one, so we can replace it ?

Reply:
and change this test case name?

Reply:
`testBasePosDeleteWriterMetrics` is removed, and `testBasePosDeleteWriter` now check the metric